src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Sura");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
+            String userInput = scanner.nextLine();
+
+            if ("bye".equalsIgnoreCase(userInput)) {
+                break;
+            }
+            if ("list".equalsIgnoreCase(userInput)) {
+                System.out.println("____________________________________________________________");
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    Task task = tasks[i];
+                    System.out.println((i + 1) + "." + task);
+                }
+                System.out.println("____________________________________________________________");
+            } else if (userInput.startsWith("mark ")) {
+                processTask(userInput, tasks, true, taskCount);
+            } else if (userInput.startsWith("unmark ")) {
+                processTask(userInput, tasks, false, taskCount);
+            } else if (userInput.startsWith("todo ")) {
+                if (taskCount >= 100) {
+                    System.out.println("You have reached the task limit. Cannot add more tasks!");
+                    continue;
+                }
+
+                String description = userInput.substring(5).trim();
+                Todo newTodo = new Todo(description);
+                tasks[taskCount] = newTodo;
+                taskCount++;
+
+                System.out.println("____________________________________________________________");
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + newTodo);
+                System.out.println("Now you have " + taskCount + " tasks in the list.");
+                System.out.println("____________________________________________________________");
+            } else if (userInput.startsWith("deadline ")) {
+                if (taskCount >= 100) {
+                    System.out.println("You have reached the task limit. Cannot add more tasks!");
+                    continue;
+                }
+
+                String content = userInput.substring(9).trim();
```

Comment:
Can use a variable or comment to explain what the significance of using 9 here is